client/src/components/scenario-dashboard/filter-top/clear-filter-button.tsx

```diff
@@ -0,0 +1,33 @@
+"use client";
+
+import { useQueryStates, parseAsString } from "nuqs";
+import { SEARCH_PARAMS } from "@/lib/constants";
+
+export default function ClearFilterButton() {
+  const [, setQuery] = useQueryStates({
+    [SEARCH_PARAMS.YEAR]: parseAsString,
+    [SEARCH_PARAMS.START_YEAR]: parseAsString,
+    [SEARCH_PARAMS.END_YEAR]: parseAsString,
+    [SEARCH_PARAMS.GEOGRAPHY]: parseAsString,
+  });
```

Comment:
Perhaps we can isolate this in a file so it's easier to find and manage. We do something similar here: https://github.com/Vizzuality/tnc-blue-carbon-cost-tool/blob/dev/client/src/containers/projects/url-store.ts